pyiron_base/jobs/job/runfunction.py

```diff
@@ -431,6 +451,53 @@ def run_job_with_runmode_srun(job):
     )
 
 
+def run_job_with_runmode_flux(job, executor, gpus_per_slot=None):
+    if not flux_available:
+        raise ModuleNotFoundError(
+            "No module named 'flux'. Running in flux mode is only available on Linux;"
+            "For CPU jobs, please use `conda install -c conda-forge flux-core`; for "
+            "GPU support you will additionally need "
+            "`conda install -c conda-forge flux-sched libhwloc=*=cuda*`"
+        )
+    if not state.database.database_is_disabled:
+        executable_template = Template(
+            """\
+#!/bin/bash
+python -m pyiron_base.cli wrapper -p {{working_directory}} -j {{job_id}}
+"""
+        )
```

Comment:
```suggestion
        executable_template = Template(
            "#!/bin/bash\n"
            "python -m pyiron_base.cli wrapper -p {{working_directory}} -j {{job_id}}\n"
        )
```

Just a suggestion, but it makes it a lot easier for folding. Same for the lines below.

Reply:
Done